scripts/standalone.mk

```diff
@@ -37,7 +37,16 @@ ifeq ($(RISCV_CMODEL),)
 RISCV_CMODEL = medany
 endif
 
-ifeq ($(LINK_TARGET),)
+ifeq ($(PROGRAM),dhrystone)
+LINK_TARGET = ramrodata
```

Comment:
Do we want to force `LINK_TARGET = ramrodata`? Obviously we recommend it, but I think this conditional should be
```
ifeq ($(PROGRAM),dhrystone)
ifeq ($(LINK_TARGET),)
```
so that users can override `LINK_TARGET` if they decide to.